cmd/run.go

```diff
@@ -28,11 +29,17 @@ func init() {
 func subMain() error {
 	ctrl.SetLogger(zap.New(zap.UseDevMode(config.development)))
 
+	var cacheFunc cache.NewCacheFunc
+	if config.namespaces != nil && len(config.namespaces) > 0 {
```

Comment:
I think `config.namespace != nil` is redundant since length of nil slice is 0.

Reply:
Removed the `config.namespaces != nil` check in this if statement. I have amended this change into the current commit.